WooCommerce/Classes/ViewRelated/Orders/Order Creation/NewOrder.swift

```diff
@@ -70,11 +76,182 @@ struct NewOrder: View {
     }
 }
 
+// MARK: Order Sections
+/// Represents the Products section
+///
+private struct ProductsSection: View {
+    let geometry: GeometryProxy
+
+    var body: some View {
+        Group {
+            Divider()
+
+            VStack(alignment: .leading, spacing: NewOrder.Layout.verticalSpacing) {
+                Text(NewOrder.Localization.products)
+                    .headlineStyle()
+
+                // TODO: Add a product row for each product added to the order
+                ProductRow()
+
+                AddButton(title: NewOrder.Localization.addProduct) {
+                    // TODO: Open Add Product modal view
+                }
+            }
+            .padding(.horizontal, insets: geometry.safeAreaInsets)
+            .padding()
+            .background(Color(.listForeground))
+
+            Divider()
+        }
+    }
+
+    /// Represent a single product row in the Product section
+    ///
+    struct ProductRow: View {
+        var body: some View {
+            AdaptiveStack(horizontalAlignment: .leading) {
+                HStack(alignment: .top) {
+                    // Product image
+                    // TODO: Display actual product image when available
+                    Image(uiImage: .productPlaceholderImage)
+                        .aspectRatio(contentMode: .fill)
+                        .frame(width: NewOrder.Layout.productImageSize, height: NewOrder.Layout.productImageSize)
```

Comment:
Not a big deal, but it would be good if the product image size could also scale with the font size. 
You can do that by using the `scaled metric` property wrapper.

https://github.com/woocommerce/woocommerce-ios/blob/4f8f59560ac35be395e3b9f30782f93a7935b9ef/WooCommerce/Classes/ViewRelated/Orders/Order%20Details/Address%20Edit/FilterListSelector.swift#L65

Reply:
TIL about the `scaled metric` property wrapper. That's perfect, thanks!

Reply:
Added this resizing in c70545f.